repos/roslyn-tools.proj

```diff
@@ -6,6 +6,7 @@
     <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
     <UsesRepoToolset>true</UsesRepoToolset>
     <BuiltToolPackageId>$(RepoToolsetPackageId)</BuiltToolPackageId>
+    <GlobalJsonFile>$(ProjectDirectory)/global.json</GlobalJsonFile>
```

Comment:
For NuGet.Config we auto-detect the basic `$(ProjectDirectory)/NuGet.Config`, thought about doing the same here to minimize the proj files? Are there some repos with global.json that we don't need to update?

Reply:
There are other repos with global.json that don't need to be updated.  They don't have sdk.version, though, but I thought it better to be explicit in this case.